app/assets/stylesheets/image_x.scss

```diff
@@ -71,3 +77,23 @@ $image-thumb-slider-border-color: #000;
 .#{$namespace}-image-x-label {
   text-align: center;
 }
+
+.#{$namespace}-frame {
+  position:absolute;
+  display:block;
+  border: 1px solid #000000;
+}
+
+.#{$namespace}-canvas {
+  position: absolute;
+  display: block;
+  border: 1px solid #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.